.github/workflows/ci.yml

```diff
@@ -33,3 +33,23 @@ jobs:
 
             - name: Run Phing
               run: composer run-script phing
+
+    phpstan:
```

Comment:
I think it's worth making PHPStan part of `phing`. Because we are getting used to just run `composer phing` to execute all quality checks.

Reply:
I'm not sure I see the value of having `phing` to run these checks.
If it's simply a matter of running "all checks at once", we can solve that with a simple composer script or a makefile.

The downside of keeping phing is that it's yet another tool in the pipeline that we need to upgrade and maintain.
I'm wondering how valuable it is to keep it.

Reply:
We could also just stop using Phing of course. I'd be fine with that. 

One way or another, I'd just like to be able to run all the ci scripts together (at least locally).

Composer scripts can achieve the same goal, with one less (dev) dependency to keep up-to-date.
See https://blog.martinhujer.cz/have-you-tried-composer-scripts/

But you're right that if we use Phing, I'd add it to the `build.xml`

If we do drop Phing here, I'd also make a ticket to remove it in other repo's later. In the meantime we can remove it on an individual basis when we work on a repo of course.

Reply:
Ah I was typing my comment while you posted yours @jdrieghe 😄

Reply:
I can also see the value of running them separately on Github Actions. I'm only wondering if we also have a `ci` script like in the article above, for running everything locally, if those two lists of checks wouldn't get out of sync easily?

Reply:
Additionally, I don't think we need to run PHP CS and phpstan against all of the supported PHP versions.

Reply:
True. So we would run the separately on Github Actions (and not always all of them for every PHP version), but also have a single script that combines them locally? I guess we can try that and deal with inconsistencies of local vs Github Actions if they pop up and if need be change strategy then.